### PR TITLE
School Library App: Add classes

### PR DESCRIPTION
**Pull request by AFakhriddinov**
**In this milestone,**

- Set linters for Ruby
- Created classes for person, student and teacher

### CODE REVIEW

File: teacher.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown')
```
Comment (Munsa1) [inline teacher.rb]: - [ ] Kindly set the parent permission to true  for this constructor as per requirement

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    super(age, true, name)
```
Comment (Munsa1) [inline teacher.rb]: - [ ] Also the super function should have (age, name, parent_permission: true) as per requirement unlike what we have

File: README.md [hunk 3]
```diff
@@ -12,7 +12,6 @@
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
   - [Presentation](#presentation)
```
Comment (Munsa1) [inline README.md]: - [ ] Kindly remove this link from the table of contents as it leads nowhere and as such is a broken link. That is the `Presentation` link

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @AFakhriddinov, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Separate Files For Each Class :heavy_check_mark: 
Correct Gitflow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (AFakhriddinov) [reply README.md]: Thanks for your feedback. I will fix this 👍

File: teacher.rb (hunk 2 above)
Comment (AFakhriddinov) [reply teacher.rb]: Thanks for your review. I will implement this  😄

**Review by ShadyShawkat — APPROVED** [review]
## Hi @AFakhriddinov, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**